tasks/Taskfile_internal.yml

```diff
@@ -0,0 +1,31 @@
+version: "3"
+
+tasks:
+  build:
+    summary: Internal build task
+    internal: true
+    cmds:
+      - cmd: echo Building MassaStation for {{.OS | default OS}}/{{.ARCH | default ARCH}}
+      - cmd: go build {{.BUILD_FLAGS}} -o {{.BIN_DIR}}/{{.APP_NAME}}{{.BIN_EXT}} ./cmd/massastation/main.go
+    vars:
+      VERSION_FLAG: '{{if ne .VERSION nil}}{{if ne .VERSION ""}}-X github.com/massalabs/station/int/config.Version={{.VERSION}}{{end}}{{end}}'
+      BUILD_FLAGS: '{{if eq .PRODUCTION "true"}}-tags production {{end}}-ldflags="{{.VERSION_FLAG}}{{if eq .PRODUCTION "true"}}{{end}}"'
```

Comment:
is this usefull?
```suggestion
      BUILD_FLAGS: '{{if eq .PRODUCTION "true"}}-tags production {{end}}-ldflags="{{.VERSION_FLAG}}"'
```

Reply:
We now use it with `-w -s` to remove debug symbols